glossary/es/index.md

```diff
@@ -1,98 +1,6 @@
 ---
-redirect_from: /es/glossary.html
-section: glossary
+redirect_from: /en/glossary.html
+layout: glossary
```

Comment:
@morchickit As far as I know, this page is not automatically created. All the auto-generated pages are in the `_site` directory, which is ignored by `.gitignore`.